app/styles/ui/_branches.scss

```diff
@@ -57,6 +57,12 @@
     padding-bottom: var(--spacing-half);
   }
 
+  .selected:focus {
+    .ci-status {
+      fill: var(--background-color);
```

Comment:
Can we make this `color: inherit` instead? I'd rather have the list item color implicitly dictate the icon color for selected+focused.

Reply:
Didn't realize/forgot that `color` can be used here.